requirements/analyzer.in

```diff
@@ -20,6 +20,14 @@ trafilatura
 # but we need to make sure lxml.html.clean is installed too
 lxml[html_clean]
 
+
+#######################################################################
+# ETL Pipeline
```

Comment:
I wonder about moving these into the ext repo, feels weird to have them here.

Reply:
It does feel a bit awkward. However, that module is not installed but rather just added to the path. There aren't requirements installed from that module currently. We could change that.

Reply:
I think the plan is to merge as-is with the idea to clean up the installation of the ext repo at some point.